docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc

```diff
@@ -92,6 +91,63 @@ WARNING: The reset will affect all widgets and the general page settings of the
 To reset all pages and remove all user created pages clear the browser's local storage for the monitoring console page.
 
 
+[[monitoring-console-page-sync]]
+==== Page Synchronisation
+Monitoring Console uses the browser's local storage to store the page configuration (which includes the widgets placed on a page).
+This means by default changes to page configurations do not affect other users.
+To make pages available to others and to start from a common base configuration pages are explicitly synchronised with a persistent server page configuration.
+
+The chosen role controls the behaviour and available options for the page synchronisation.
+It is the _Administrator_'s role that is meant to manage the persistent server page configuration.
+The _Administrator_ pushes local changes to the server to update the base configuration.
+_User_s and _Guest_s pull the server configuration to update their local configuration with the base.
```

Comment:
I'm not sure if it's just the GitHub preview, but it doesn't like this italics syntax despite working elsewhere. Best guess is the no space between the closing `_` and the `s`

Reply:
You're right, @Pandrex247, it now looks like this in Antora: ![image](https://user-images.githubusercontent.com/2195988/82217764-31fc3380-991b-11ea-86d8-e9406b3c8602.png)

@jbee, if you need to format only a part of a word, you need to double the underscores. This is called "unconstrained" formatting in the Asciidoc documentation: https://asciidoctor.org/docs/asciidoc-writers-guide/#quoted-text

This should work as expected:

```suggestion
__User__s and __Guest__s pull the server configuration to update their local configuration with the base.
```